test/cql-pytest/test_shedding.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+#############################################################################
+# Tests for the shedding mechanisms in the CQL layer
+#############################################################################
+
+import pytest
+import re
+from cassandra.protocol import InvalidRequest
+from util import unique_name, random_string
+
+
+@pytest.fixture(scope="session")
```

Comment:
Two thoughts for the future (no need to do it now):
1. We should have one or several text fixtures that all tests can use if they don't really care about the schema, like is the case here, instead of every test file needing to add test fixtures.
2. We should change "session" scope fixtures to "module" scope. So the tables created by a test file get deleted after the test file - not just when all the test suite is done.